site/source/tutorials/9001_dojo1_dijits/index.md

```diff
@@ -0,0 +1,187 @@
+{% endaside %}
+
+In the previous step, we need to include a configuration for our Dojo 1 loader.  For this tutorial, we only need to ensure that the Dojo 1 loader works in an asynchronous fashion, because the Dojo 1 loader will default to synchronous operation, which will perform slower and cause issues with our application.  The file could be named anything, but in this tutorial, we are naming it `/src/dojo1.cfg.js`.
+
+{% instruction 'Create a `/src/dojo1.cfg.js` with the following content:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/dojo1.cfg.js' %}
+
+{% task 'Ensure the application builds.' %}
+
+At this point, you should be able to build your application.
+
+{% instruction 'Run `dojo build` in the project root.' %}
+
+{% aside 'Reminder' %}
+You can now use `dojo build -w` to continue to develop at this point.
+{% endaside %}
+
+You should see a rather long output indicating lots of files have been output.
+
+{% section %}
+
+## Dealing with Dijits
+
+In order for Dijits to be integrated into a Dojo 2 application, they need to _wrapped_ so that they can be managed by the Dojo 2 widgeting system.  As discussed in previous steps, there is a function named `DijitWrapper` which will _convert_ a Dojo 1 Dijit constructor function into something that behaves like a Dojo 2 widget class.  When used with the Dojo 1 typings, you will also get code intellisense when configuring your Dijits.
+
+The `DijitWrapper` function takes two arguments.  The first is the Dojo 1 `declare` class.  The second is an optional string, which represents that tag name that should be used when the Dojo 2 widgeting system needs to create a _DOM stub_ to attach the Dijit instance to.  It defaults to `div` if not supplied.
+
+{% task 'Wrap Dijits.' %}
+
+{% aside 'Information' %}
+All Dojo 1 modules are AMD modules without default exports.  Because of this, you have to import them using the `import * as Dijit from 'module/id';` syntax.
+{% endaside %}
+
+We recommend that any Dijits that you wrap, that you create them as individual modules in `/src/dijit` in the same path structure they are located in the source `dijit` package.
+
+{% instruction 'Create `/src/dijit/Fieldset.ts` with the following content:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/dijit/Fieldset.ts' %}
+
+{% instruction 'Create `/src/dijit/form/Button.ts` with the following content:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/dijit/form/Button.ts' %}
+
+{% instruction 'Create `/src/dijit/form/TextBox.ts` with the following content:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/dijit/form/TextBox.ts' %}
+
+{% task 'Importing Dijits.' %}
+
+Now that you have wrapped Dijits that you want to use, we need to import them into the Dojo 2 which will contain them.
```

Comment:
that you want to use, we need to import them into the Dojo 2 which will contain them -> for use, we need to import them into their containing Dojo 2 widgets or modules

Reply:
you don't import things into widgets, you import things into modules